gatsby/src/components/link-list/link-list.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import LinkStyled from '@/components/link-styled';
+
+import useMobile from '@/hooks/useMobile';
+
+interface Props {
+  linkList: any;
```

Comment:
it is definitely not any, and I cannot see `linkList` in the component
```
interface Props {
  fallbackClass?: string,
  links: Array<{ title: string, path: { alias: string } }>
}
```

Reply:
edited